app/models/recruitment_cycle.rb

```diff
@@ -10,6 +10,10 @@ class RecruitmentCycle < ApplicationRecord
 
   validates :year, presence: true
 
+  scope :ordered_by_current_first, lambda {
+    order(Arel.sql("CASE WHEN year = '#{Settings.current_recruitment_cycle_year}' THEN 0 ELSE 1 END, year DESC"))
```

Comment:
Relying on `Settings.current_recruitment_cycle_year` means that when the new cycle starts we will have to change the yaml file and do a deployment.

🤔 🤔 🤔 

Would be good for us to consider to remove the usage of `Settings.current_recruitment_cycle_year` on any checks and use from the recruitment cycle object/timetable we will about to create 

As for this PR might be fine but we will have to revisit this code.

@Nitemaeric / @gms-gs about the above what are your thoughts?